qemu/tests/block_multifunction.py

```diff
@@ -110,9 +110,10 @@ def get_image_device(qdev, img_name):
     disk_op_cmd = params.get("disk_op_cmd")
     session = vm.wait_for_login()
 
-    q35 = params['machine_type'] == 'q35'
-    dev_slot = 0 if q35 else 9
-    parent_bus = 'pcie_extra_root_port_0' if q35 else 'pci.0'
+    if "q35" or "arm64" in params['machine_type']:
```

Comment:
s/arm64/arm64-pci

Reply:
```
machine_type = 'arm64-mmio:virt'
if "q35" or "s/arm64/arm64-pci" in machine_type:
    print('b')
    
b
```

If we want to avoid 'arm64-mmio' should we use `arm64-pci` retrieval directly